notebooks/summit/AuxTel/SITCOM-476/SITCOM-476_LargeDonuts.ipynb

```diff
@@ -0,0 +1,761 @@
+{
```

Comment:
<p>Set the <code>program</code> key as well please. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_isotuela/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
👍